experiments/Atmos_GCM/heldsuarez.jl

```diff
@@ -1,61 +1,89 @@
+using Distributions: Uniform
 using LinearAlgebra
 using StaticArrays
+using Random: rand
 using Test
 
 using CLIMA
 using CLIMA.Atmos
 using CLIMA.GenericCallbacks
 using CLIMA.ODESolvers
+using CLIMA.ColumnwiseLUSolver: ManyColumnLU
 using CLIMA.Mesh.Filters
 using CLIMA.Mesh.Grids
 using CLIMA.MoistThermodynamics
 using CLIMA.PlanetParameters
 using CLIMA.VariableTemplates
 
-const p_ground = MSLP
-const T_initial = 255
-const domain_height = 30e3
 
-function init_heldsuarez!(bl, state, aux, coords, t)
-    global p_ground, T_initial
-
-    FT = eltype(state)
+const pressure_ground = MSLP
+const T_init = 300.0 # unit: Kelvin 
```

Comment:
It doesn't matter much what you choose as the initial temperature, but `T_init = 255` will probably equilibrate a bit faster (because the stratosphere, where relaxation times are longer, will end up with a temperature around 200 K).